src/webgpu/api/validation/state/device_lost/destroy.spec.ts

```diff
@@ -500,6 +505,43 @@ Tests creating query sets on destroyed device.
     }, awaitLost);
   });
 
+g.test('importExternalTexture')
+  .desc(
+    `
+Tests import external texture on destroyed device. Tests valid combinations of:
+  - Various valid source type
+  `
+  )
+  .params(u =>
+    u
+      .combine('sourceType', ['VideoElement', 'VideoFrame'] as const)
+      .beginSubcases()
+      .combine('awaitLost', [true, false])
+  )
+  .fn(async t => {
+    const { awaitLost, sourceType } = t.params;
+
+    const videoElement = getVideoElement(t, 'four-colors-vp9-bt601.webm');
+    if (!('requestVideoFrameCallback' in videoElement)) {
+      t.skip('HTMLVideoElement.requestVideoFrameCallback is not supported');
+    }
+
+    let source: HTMLVideoElement | VideoFrame;
+    await startPlayingAndWaitForVideo(videoElement, async () => {
+      source =
+        sourceType === 'VideoFrame'
+          ? await getVideoFrameFromVideoElement(t, videoElement)
+          : videoElement;
+
+      await t.executeAfterDestroy(() => {
+        t.device.importExternalTexture({
+          /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
+          source: source as any,
+        });
```

Comment:
nit: It would probably be good to keep something that uses the texture, like a createBindGroup call

Reply:
Done.